src-ui/src/app/components/manage/settings/settings.component.ts

```diff
@@ -733,6 +775,68 @@ export class SettingsComponent
     })
   }
 
+  editSSOGroup(ssoGroup: PaperlessSSOGroup) {
+    var modal = this.modalService.open(SsoGroupEditDialogComponent, {
+      backdrop: 'static',
+      size: 'lg',
```

Comment:
```suggestion
```

Doesnt need to be large =)

Reply:
In the settings, all the modals are lg or xl, I just wanted to keep it consistent, I'll change that for my next push

Reply:
Default is 'md', so all of the calls that don't specify size are smaller. Other settings have many fields as opposed to just the 2 here. Thanks